smriprep/workflows/surfaces.py

```diff
@@ -56,7 +56,7 @@
 from smriprep.interfaces.surf import MakeRibbon
 from smriprep.interfaces.workbench import SurfaceResample
 
-from ..data import load_resource
+from ..data import load
```

Comment:
Maybe

```suggestion
from .. import data
```

And use `data.load()` below? I feel like a bare `load` variable is really ambiguous. I almost want to say `import smriprep` and use `smriprep.data.load()`.

Reply:
Agreed, I think it's much cleaner.

I went with importing and aliasing `smriprep.data.load` to be used like:
`smriprep.load_data(...)`